astropy/utils/data.py

```diff
@@ -1343,26 +1344,6 @@ def _get_download_cache_locs():
     return datadir, shelveloc
 
 
-def _open_shelve(shelffn, withclosing=False):
-    """
-    Opens a shelf file.  If `withclosing` is  True, it will be opened with closing,
-    allowing use like:
-
-        with _open_shelve('somefile',True) as s:
-            ...
-    """
-    import shelve
-
-    shelf = shelve.open(shelffn, protocol=2)
-
-    if withclosing:
-        return contextlib.closing(shelf)
-    else:
-        return shelf
-
-
-# the cache directory must be locked before any writes are performed.  Same for
```

Comment:
Just not sure if this comment line should be removed, as it is not attached to the function that was removed but the one after that that is still here.

Reply:
ok, I'll put this back once travis passed.